data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (Id,Name,Date_of_birth,Escape_attempts,Neutered,Weight_kg) VALUES (1, 'Agumon', (to_date('02/03/2020','mm/dd/yyyy')), 0, True, 10.23 );
+INSERT INTO animals (Id,Name,Date_of_birth,Escape_attempts,Neutered,Weight_kg) VALUES (2, 'Gabumon', (to_date('11/15/2018','mm/dd/yyyy')), 2, True, 8 );
+INSERT INTO animals (Id,Name,Date_of_birth,Escape_attempts,Neutered,Weight_kg) VALUES (3, 'Pikachu', (to_date('01/07/2021','mm/dd/yyyy')), 1, False, 15.04 );
+INSERT INTO animals (Id,Name,Date_of_birth,Escape_attempts,Neutered,Weight_kg) VALUES (4, 'Devimon', (to_date('05/12/2017','mm/dd/yyyy')), 5, True, 11 );
```

Comment:
- [ ] In the insertion query, we don't need to specify the id we are going to assign the data because it is auto generated, kindly remove the ID in the insert. Imagine once you have more than 1000 Ids

Reply:
Than you for the remark. Done

Reply:
Well done